src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    {
+      age && <p className="Person__age">{`I am ${age}`}</p>
+    }
```

Comment:
let's format it properly
```suggestion
    {age && (
      <p className="Person__age">
        {`I am ${age}`}
      </p>
    )}
```